lib/PuppeteerSharp/CustomQueriesManager.cs

```diff
@@ -155,21 +190,23 @@ internal void RegisterCustomQueryHandler(string name, CustomQueryHandler queryHa
 
         internal (string UpdatedSelector, PuppeteerQueryHandler QueryHandler) GetQueryHandlerAndSelector(string selector)
         {
-            var customQueryHandlerMatch = _customQueryHandlerParserRegex.Match(selector);
-            if (!customQueryHandlerMatch.Success)
-            {
-                return (selector, _defaultHandler);
-            }
-
-            var name = customQueryHandlerMatch.Groups["query"].Value;
-            var updatedSelector = customQueryHandlerMatch.Groups["selector"].Value;
+            var handlers = _internalQueryHandlers.Concat(_queryHandlers);
 
-            if (!_queryHandlers.TryGetValue(name, out var queryHandler))
+            foreach (var kv in handlers)
```

Comment:
Use 4 spaces instead of 2 🧹

Reply:
ohhh just merged! Let's create a follow up PR

Reply:
I'll create a PR right away

Reply:
I have one ready to go :)